setup.py

```diff
@@ -156,7 +149,7 @@ def find_package_data(where=".", package="", exclude=standard_exclude,
     long_description=LONG_DESCRIPTION,
     platforms=['any'],
     classifiers=CLASSIFIERS,
-    install_requires=["boto3>=1.0.0", "pytz>=2016.10", "future>=0.16.0", "django>1.10"],
+    install_requires=["boto3>=1.0.0", "pytz>=2016.10", "future>=0.16.0", "django>=2.2"],
```

Comment:
On line 150 above, it still has `python_requires='>=2.7'` - could you update that, as well?

Reply:
No problem, I missed it.